src/Analysis/Ast/Test/TypingTests.cs

```diff
@@ -64,11 +64,57 @@ import io
 ";
             var analysis = await GetAnalysisAsync(code);
             analysis.Should().HaveVariable("T")
-                .Which.Value.Should().HaveDocumentation("TypeVar('T', TextIOWrapper)");
+                .Which.Value.Should().HaveDocumentation("TypeVar('T', bound=io.TextIOWrapper)");
             analysis.Should().HaveVariable("T").OfType(typeof(IGenericTypeDefinition));
         }
 
 
+        [TestMethod, Priority(0)]
+        public async Task TypeVarCovariantDocCheck() {
+            const string code = @"
+from typing import TypeVar
+
+T = TypeVar('T', str, int, covariant=True)
+";
+            var analysis = await GetAnalysisAsync(code);
+            analysis.Should().HaveVariable("T")
+                .Which.Value.Should().HaveDocumentation("TypeVar('T', str, int, covariant=True)");
+            analysis.Should().HaveVariable("T").OfType(typeof(IGenericTypeDefinition));
```

Comment:
`analysis.Should().HaveVariable("T").Which.Should().BeAssignableTo<IGenericTypeDefinition>();` or `analysis.Should().HaveVariable("T").OfType<IGenericTypeDefinition>()`

Reply:
Or `analysis.Should().HaveVariable("T").Which.IsGeneric().Should().BeTrue()` but it is a bit broader check.

Reply:
I'd actually write fluent extension so we could write `analysis.Should().HaveGenericVariable("T")`. @AlexanderSher?

Reply:
I'll try to do the fluent assertions